diff --git a/dist/shared/test-utils.RVsKJA_7.mjs b/dist/shared/test-utils.RVsKJA_7.mjs
index c20383a7ecb7a7da7e5b4876c4451f7f6315ea06..fc05193730696591c00eb9b6b91a950b572066db 100644
--- a/dist/shared/test-utils.RVsKJA_7.mjs
+++ b/dist/shared/test-utils.RVsKJA_7.mjs
@@ -115,7 +115,8 @@ async function buildFixture() {
 }
 
 async function setupBun(hooks) {
-  const bunTest = await import('bun:test');
+  const specifier = 'bun:test';
+  const bunTest = await import(specifier);
   hooks.ctx.mockFn = bunTest.mock;
   bunTest.beforeAll(hooks.beforeAll, hooks.ctx.options.setupTimeout);
   bunTest.beforeEach(hooks.beforeEach);
